graphql2/schema.graphql

```diff
@@ -655,6 +657,13 @@ input RotationSearchOptions {
   favoritesFirst: Boolean = false
 }
 
+input NextRotationHandoffTimesInput {
+  start: ISOTimestamp!
+  timeZone: String!
+  hours: Int!
```

Comment:
I feel like `end: ISOTimestamp!` makes more sense instead of an `hours` duration, as this would match how we query for shifts and overrides within gql

Reply:
I think this is a misunderstanding, this is the shift length not a duration for the query. Maybe we should call it `shiftLengthHours`?

Reply:
From an API standpoint (black box), I'm not sure why I would need to provide the rotation's shift length when asking for the next handoff times. Wouldn't `count` be sufficient?